DOL.WHD.Section14c.Test/Business/ErrorLogRepositoryTest.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using DOL.WHD.Section14c.Log.Repositories;
+using DOL.WHD.Section14c.Test.RepositoryMocks;
+using DOL.WHD.Section14c.Log.Controllers;
+using DOL.WHD.Section14c.Log.Models;
+using System.Linq;
+using System.Web.Http.Results;
+using DOL.WHD.Section14c.Log.LogHelper;
+
+namespace DOL.WHD.Section14c.Test.Business
+{
+    [TestClass]
+    public class ErrorLogRepositoryTest
```

Comment:
Same comment as previous - this is testing `ErrorLogsController`, not `ErrorLogRepository`